opm/simulators/flow/Transmissibility_impl.hpp

```diff
@@ -725,35 +674,32 @@ removeNonCartesianTransmissibilities_(bool removeAll)
 template<class Grid, class GridView, class ElementMapper, class CartesianIndexMapper, class Scalar>
 void Transmissibility<Grid,GridView,ElementMapper,CartesianIndexMapper, Scalar>::
 applyAllZMultipliers_(Scalar& trans,
-                      unsigned insideFaceIdx,
-                      unsigned outsideFaceIdx,
-                      unsigned insideCartElemIdx,
-                      unsigned outsideCartElemIdx,
+                      const FaceInfo& inside,
+                      const FaceInfo& outside,
                       const TransMult& transMult,
                       const std::array<int, dimWorld>& cartDims)
 {
-    if(grid_.maxLevel()> 0) {
-                OPM_THROW(std::invalid_argument, "MULTZ not support with LGRS, yet.");
+    if (grid_.maxLevel() > 0) {
+        OPM_THROW(std::invalid_argument, "MULTZ not support with LGRS, yet.");
```

Comment:
@akva2 so what we do not support yet is MULTZ (for CpGrid with LGRs), throwing when calling applyAllZMultipliers(...). applyMultipliers does not throw for CpGrid with Lgrs. Then it can/must be a bug.

Reply:
As far as I can see, the only difference here is that we are on the boundary.